### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by Farinoxa**

### CODE REVIEW

File: src/modules/shared/layouts/AppLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Footer from '../Footer/Footer';
```
Comment (des1nteresado) [inline src/modules/shared/layouts/AppLayout.jsx]: Header.jsx, Footer.jsx => index.jsx

File: src/modules/shared/layouts/AppLayout.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import Header from '../Header/Header';
+
+// eslint-disable-next-line react/prop-types
```
Comment (des1nteresado) [inline src/modules/shared/layouts/AppLayout.jsx]: Let's use proptypes

File: package.json [hunk 3]
```diff
@@ -9,6 +9,8 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "material-ui-search-bar": "^1.0.0",
```
Comment (des1nteresado) [inline package.json]: please remove unused libs

File: src/modules/shared/Footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Footer.css';
+
+function Footer() {
+  return (
+    <div className="Footer_Container">
```
Comment (des1nteresado) [inline src/modules/shared/Footer/Footer.jsx]: camelCase => footerContainer

File: src/modules/shared/Chips/Chips.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Chip, Grid } from '@material-ui/core';
+
+function Chips() {
+  return (
+    <Grid container>
+      <h2>Related Topics</h2>
+
+      <Chip clickable label="Java Script" />
```
Comment (des1nteresado) [inline src/modules/shared/Chips/Chips.jsx]:
```
const chipLabels = ['JavaScript', 'Health', 'Reactjs', ...];

chipLabels.map((label) => <Chip key={label} clickable label={label} />)
```

File: src/modules/Home/components/Content/Content.jsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+              <Article
+                userAvatar={article.userAvatar}
+                userName={article.userName}
+                title={article.title}
+                description={article.description}
+                newsPicture={article.newsPicture}
+              />
+            </Grid>
+            <Divider orientation="vertical" flexItem />
+            <Grid item md={2}>
```
Comment (des1nteresado) [inline src/modules/Home/components/Content/Content.jsx]: Ussually grid container contains 12 elemets. 
In your case 9 + 2 = 11

File: src/modules/Home/components/Content/Content.jsx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        <Paper
+          elevation={0}
+          variant="outlined"
+          className="themeCoverPaper"
+          style={{ backgroundImage: `url(https://source.unsplash.com/random)` }}
+        >
+          <Container fixed>
+            <div className="themeCoverOverlay" />
+            <Grid Container>
+              <Grid item>
```
Comment (des1nteresado) [inline src/modules/Home/components/Content/Content.jsx]: do you really need grid container with one item?

File: src/modules/Home/components/Content/Content.jsx (hunk 6 above)
Comment (Farinoxa) [reply src/modules/Home/components/Content/Content.jsx]: if I follow 12 elements, then the block with the chips is shifted down. I think the reason is the divider

**Comment by Farinoxa** [comment]
fixed the code according to the comments

File: src/modules/shared/Chips/index.jsx [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    'JavaScript',
+    'Health',
+    'Reactjs',
+    'Food',
+    'Front End Development',
+    'Nodejs',
+  ];
+
+  const chip = chipLabels.map((label) => (
+    <Chip key={label} clickable label={label} />
```
Comment (des1nteresado) [inline src/modules/shared/Chips/index.jsx:15]: `
<Chip key={label} clickable label={label} sx={{marginBottom: '10px', marginRight: '5px' }} />
`

`
<Chip key={label} clickable label={label} className="chipTag" />
`

**Comment by Farinoxa** [comment]
everything all right

**Pull request closed without merging**